### ISSUE TITLE
Any files OK to delete after case creation?

### ISSUE THREAD

**Issue by evalieungh** [NorESMhub/noresm-land-sites-platform#157]
With the default setup, the whole package of the LSP takes up quite a bit of space. I'm trying to run on a virtual machine on NREC, but it only has 20GB of space so I quickly run out of space when trying to run a long simulation (1000 yrs).

Are there any files I could delete manually to save space, after I have created a case but before it is run?

**Comment by lasseke** [comment]
With the new setup, there is not much you can do I think, unfortunately. The input data files + containers will get you close to the 20 GB limit already. One thing that might help is to disable all the history variables you do not need, e.g. by disabling h0 and only adding the ones of interest to h1. But I think a better strategy is to contact NREC and ask for more storage!

**Comment by evalieungh** [comment]
Hmm, I'm going to try their 'Volumes' thing. It says it can be attached to an instance so maybe I can get more storage that way! <https://docs.nrec.no/manage-volumes.html>

**Comment by evalieungh** [comment]
Think I got it now. Helpful person at USIT gave these instructions:

1. Make a file system on the disk:
```
sudo mkfs.xfs /dev/sdb
```
2. Make a mount point (folder to mount volume in) e.g. called "data" directly under root:
```
sudo mkdir /data
```
3. Mount the volume:
```
sudo mount /dev/sdb /data
```
This is not persistent after reboot.

**Comment by evalieungh** [comment]
I moved the repo over to ../../../data/noresm-land-sites-platform from the home folder and did `docker compose up` again there. Since the screen is also from the root folder it seems like it works fine. I'm running a case I submitted before this fix, and it seems to be doing fine so far. 

If this works I can add a sentence about this option to the documentation.

**Comment by emiliano-gelati** [comment]
Hi @evalieungh,

Using volumes was on my to-do list, but I have never tried. Great that you made it work. As the mounting is not persistent after reboot, a workaround could be adding [your commands](https://github.com/NorESMhub/noresm-land-sites-platform/issues/157#issuecomment-1324807498) to .bashrc in the virtual machine. However, there may be cleaner ways of doing it.

On the other hand, I could not find scripts and instructions on how to set up a virtual machine on NREC. Therefore I have created a pull request to restore those files (https://github.com/NorESMhub/noresm-land-sites-platform/pull/158). I could not add reviewers, perhaps because I no longer have write permissions on the repository.

**Comment by evalieungh** [comment]
Thanks, Emiliano! 
According to the USIT person, a way of making it persistent is to add the following line to /etc/fstab:
`/dev/sdb  /data  xfs  default  0 0` (in my case sdb is the name of my volume and it's mounted in the data dir)
I tried this but wasn't able to edit the file. I got some read-only error that prevented saving even though I tried to give myself write permission with chmod +w.

I think we moved the repository from a different account, so our old history is missing. I'll take a look at the PR and involve the others too. If you want we will give you more permissions with the repo !

**Comment by lasseke** [comment]
@emiliano-gelati the old NREC scripts are still available within this repository, under the `archive` tag: https://github.com/NorESMhub/noresm-land-sites-platform/tree/archive/.machine/NREC_VM_setup

In my opinion, it is better to keep it this way (relating to #158) as this information is very use-case specific and not very relevant for people outside the Norwegian university networks. On the other hand, I think a lot of people, who might not even intend to use the platform, could benefit from these tutorials and scripts. They save redundant questions and work, so they should be easier to access! How about we approach the people in charge of the model development documentation in our department or create a new space for shared modelling resources at UiO?

**Comment by emiliano-gelati** [comment]
I agree with both of you. I was not aware of https://github.com/NorESMhub/noresm-land-sites-platform/tree/archive/

**Comment by ka7eh** [comment]
> Thanks, Emiliano! According to the USIT person, a way of making it persistent is to add the following line to /etc/fstab: `/dev/sdb /data xfs default 0 0` (in my case sdb is the name of my volume and it's mounted in the data dir) I tried this but wasn't able to edit the file. I got some read-only error that prevented saving even though I tried to give myself write permission with chmod +w.
> 
> I think we moved the repository from a different account, so our old history is missing. I'll take a look at the PR and involve the others too. If you want we will give you more permissions with the repo !

Did you try with `sudo`? You can only edit that file as root.

Also, it's better to use the device id instead of its name because name might change but id will stay the same.
You can get your volume id by running `lsblk -f`. Then use the value under `UUID` in the following line:

`<UUID> /data    xfs    default    0    0`

**Comment by evalieungh** [comment]
Thanks! I did try with sudo, and was able to enter editing mode (`i`) but then it didn't want to save the changes. I can try again later today.

**Comment by evalieungh** [comment]
@emiliano-gelati I wonder if the best thing for now is to make your own repository so the scripts become available and your work is highlighted. Then we can make a reference to that repo in the documentation/user guide.

**Comment by emiliano-gelati** [comment]
For the time being it is fine with a pointer to [the archived scripts](https://github.com/NorESMhub/noresm-land-sites-platform/tree/archive/.machine/NREC_VM_setup), I think.
